docs/content/documentation/systems/MeshModifiers/framework/SubdomainBoundingBox.md

```diff
@@ -1,6 +1,7 @@
-<!-- MOOSE Documentation Stub: Remove this when content is added. -->
-
 # SubdomainBoundingBox
+
+This MeshModifier takes a user specified axis aligned box specified in terms of extreme coordinates (lower left and upper right), and changes all element subdomain IDs either inside xor outside the box to the specified ID.
+
```

Comment:
xor -> or

Reply:
Ha, I actually meant exclusive or, because in this case it is exclusive. You can't do both at the same time. I can reword so that somebody other than a geek can understand.